lib/screen/home_screen.dart

```diff
@@ -175,10 +175,14 @@ class _HomeScreenState extends State<HomeScreen> with CommandHandler {
     // Get a specific camera from the list of available cameras.
     final firstCamera = cameras.first;
 
+    //Define the source of the camera like the screen
+    // from where the camera is called (note or book)
+    final cameraSource = CameraSource.book;
+
     Navigator.push(
       context,
       MaterialPageRoute(
-          builder: (context) => TakePictureScreen(camera: firstCamera)),
+          builder: (context) => TakePictureScreen(camera: firstCamera, cameraSource: cameraSource)),
```

Comment:
This is good but I think you should use the `cameraSource` to prevent the display of the cropper, I think we don't need the cropper for the book cover image.

Reply:
Ok got it